package.json

```diff
@@ -2,12 +2,12 @@
   "name": "protosaurus",
   "version": "1.0.0",
   "private": true,
-  "workspaces":[
-    "website/*",
+  "workspaces": [
+    "website",
     "packages/*"
   ],
   "engines": {
-    "node": "16.13.2",
+    "node": ">=16.13",
```

Comment:
Turns out it was strict :grimacing: I was getting this:

```
error protosaurus@1.0.0: The engine "node" is incompatible with this module. Expected version "16.13.2". Got "16.13.0"
error Found incompatible module.
```

Let's settle for a minimum version instead, if that's OK?

Reply:
This is fine. But to provide a hermetic build, do we need to update the one in Tools.mk instead? Since I seldom to use node from system's global, but rather from .cache/tools/prepackage/bin/node.